frontend/app/(profile)/profile/projects/[project_id]/page.tsx

```
@@ -0,0 +1,305 @@
+
+          {/* Project Details Grid */}
+          <div className="grid grid-cols-1 md:grid-cols-2 gap-8">
+            <div className="bg-white p-5 rounded-lg shadow-sm border border-gray-100 hover:shadow-md transition-shadow">
+              <h3 className="text-xl font-semibold mb-4 text-gray-800 border-b pb-2">
+                Project Details
+              </h3>
+              <div className="space-y-3">
+                <p className="flex justify-between items-center">
+                  <span className="font-medium text-gray-600">Budget:</span>{" "}
+                  <span className="text-green-600 font-semibold bg-green-50 px-3 py-1 rounded-full">
+                    ৳{parseFloat(project.budget).toFixed(2)}
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)